flocker/node/agents/ebs.py

```diff
@@ -29,6 +32,45 @@
 CLUSTER_ID_LABEL = u'flocker-cluster-id'
 ATTACHED_DEVICE_LABEL = u'attached-device-name'
 BOTO_NUM_RETRIES = u'10'
+BOTO_DEBUG_LEVEL = u'1'
+
+# Scaffolding for logging Boto client and server exceptions
+# via Eliot.
+CODE = Field.for_types("code", [int], u"The HTTP response code.")
+MESSAGE = Field.for_types(
+    "message", [bytes, unicode],
+    u"A human-readable error message given by the response.",
+)
+DETAILS = Field.for_types("details", [dict], u"Extra details about the error.")
```

Comment:
Oh, I see this is exactly how @exarkun did it for OpenStack....I guess he knows what he's doing....so fine.